### ISSUE TITLE
functions don't work

### ISSUE THREAD

**Issue by elitan** [nhost/cli-go#21]
```js
import { Request, Response } from 'express';
import { getUser } from './utils/utils';

const handler = (req: Request, res: Response) => {
  console.log('test function');

  console.log(req.get('content-type'));
  console.log(req.method);
  console.log(process.env);

  res.status(200).send('OK');
};

export default handler;
```

```
curl http://localhost:1337/v1/functions/test
```

Result

```
2021/09/14 12:32:09 http: proxy error: unsupported protocol scheme ""
```

My Nhost version:

```
╰─➤  nhost version
[INFO] [amd64] [darwin] v1.0-internal-26
[INFO] You have the latest version. Hurray!
```

**Comment by mrinalwahal** [comment]
Fixed in v1.0-internal-28. Please close this issue once you have tested it.

**Comment by elitan** [comment]
all OK
